src/utils/durableUtils.ts

```diff
@@ -229,48 +238,46 @@ export namespace netheriteUtils {
-    //     const hasValidEventHubName: boolean = !!eventHubName && eventHubName !== netheriteUtils.defaultNetheriteHubName;
+        // Found a valid connection in debug mode, no need to proceed further
+        if (!!eventHubsConnection && !!eventHubName && !options?.setConnectionForDeploy) {
+            return;
+        }
 
-    //     const wizardContext: IEventHubsConnectionWizardContext = Object.assign(context, { projectPath });
-    //     const promptSteps: AzureWizardPromptStep<IEventHubsConnectionWizardContext>[] = [];
-    //     const executeSteps: AzureWizardExecuteStep<IEventHubsConnectionWizardContext>[] = [];
+        const wizardContext: IEventHubsConnectionWizardContext = Object.assign(context, { projectPath });
+        const promptSteps: AzureWizardPromptStep<IEventHubsConnectionWizardContext>[] = [];
+        const executeSteps: AzureWizardExecuteStep<IEventHubsConnectionWizardContext>[] = [];
 
-    //     if (hasEventHubsConnection && hasValidEventHubName && options?.setConnectionForDeploy) {
-    //         Object.assign(context, { eventHubConnectionForDeploy: eventHubsConnection });
-    //     } else {
-    //         promptSteps.push(new EventHubsConnectionPromptStep({ preSelectedConnectionType: options?.preSelectedConnectionType, suppressSkipForNow: true }));
-    //         executeSteps.push(new EventHubsConnectionExecuteStep(options?.setConnectionForDeploy));
-    //     }
+        if (!!eventHubsConnection && !!eventHubName && options?.setConnectionForDeploy) {
+            // Found a valid connection in deploy mode
+            Object.assign(context, { eventHubConnectionForDeploy: eventHubsConnection });
+        } else {
+            promptSteps.push(new EventHubsConnectionPromptStep({ preselectedConnectionType: options?.preselectedConnectionType }));
+            executeSteps.push(new EventHubsConnectionExecuteStep(options?.setConnectionForDeploy));
+        }
 
-    //     if (!hasValidEventHubName) {
-    //         promptSteps.push(new NetheriteEventHubNameStep());
-    //     }
+        if (!eventHubName) {
+            promptSteps.push(new NetheriteEventHubNameStep());
+        }
 
-    //     executeSteps.push(new NetheriteConfigureHostStep());
+        executeSteps.push(new NetheriteConfigureHostStep());
 
-    //     const wizard: AzureWizard<IEventHubsConnectionWizardContext> = new AzureWizard(wizardContext, {
-    //         promptSteps,
-    //         executeSteps
-    //     });
+        const wizard: AzureWizard<IEventHubsConnectionWizardContext> = new AzureWizard(wizardContext, {
+            promptSteps,
+            executeSteps
+        });
 
-    //     await wizard.prompt();
-    //     await wizard.execute();
-    // }
+        await wizard.prompt();
+        await wizard.execute();
+    }
 
     export function getDefaultNetheriteTaskConfig(hubName?: string): INetheriteTaskJson {
         return {
-            hubName: hubName || defaultNetheriteHubName,
+            hubName: hubName || '',
```

Comment:
Nah it's just looking for a valid name, but not that specific name.  The only thing that needs an exact match is the emulator string `Memory` or `MemoryF`.  

I think I originally had that in there for the `skipForNow` scenario because I didn't want to leave the required field blank.  Since the check on `debug` and `deploy` is more robust now, we don't really need that placeholder there anymore.